rtv/subreddit_page.py

```diff
@@ -74,7 +74,10 @@ def prompt_subreddit(self):
 
         name = self.term.prompt_input('Enter page: /')
         if name is not None:
-            self.refresh_content(order='ignore', name=name)
+            if not '/comments/' in name:
+                self.refresh_content(order='ignore', name=name)
+            else:
+                self.open_submission(url='https://www.reddit.com/r/'+name)
```

Comment:
What's the intended format for typing in comment urls? The first time I tried this I went to https://www.reddit.com/r/pics/comments/571dw3 by typing
```
/r/pics/comments/571dw3
```
And it failed because it took me to
```
https://www.reddit.com/r/r/pics/comments/571dw3
```

Also, interestingly you don't need to put the subreddit name in the url so I could have also done https://www.reddit.com/comments/571dw3
```
/comments/571w3
```
That case also would also fail because ``name`` doesn't keep the slash at the start.